src/main/resources/db/migration/V1.1734004906__Fill_lectors_data.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO lectors (id, email, first_name, last_name) VALUES
```

Comment:
another (probably more appropriate) way to populate database is via spring bootstrap script: https://stackoverflow.com/a/38047021
this way it won't be part of migration